packages/flame/lib/src/widgets/nine_tile_box.dart

```diff
@@ -60,20 +61,25 @@ class NineTileBoxWidget extends StatelessWidget {
   /// A builder function that is called while the loading is on the way
   final WidgetBuilder? loadingBuilder;
 
-  NineTileBoxWidget({
+  const NineTileBoxWidget({
     required Image image,
     required this.tileSize,
     required this.destTileSize,
     this.width,
     this.height,
     this.child,
     this.padding,
-    this.errorBuilder,
-    this.loadingBuilder,
     Key? key,
-  })  : _imageFuture = (() => Future.value(image)),
+  })  : _imageFuture = image,
+        errorBuilder = null,
+        loadingBuilder = null,
         super(key: key);
 
+  /// loads image from asset [path] and renders it as Widget.
+  /// it has [loadingBuilder] while the image from [path] loads image
+  /// To render without loading or gapless playback when the [path] change,
+  /// consider load an image from somewhere and directly pass it to
+  /// default constructor
```

Comment:
```suggestion
  /// Loads image from the asset [path] and renders it as a widget.
  ///
  /// It will use the [loadingBuilder] while the image from [path] is loading.
  /// To render without loading, or when you want to have a gapless playback 
  /// when the [path] value changes, consider loading the image beforehand 
  /// and direct pass it to the default constructor.
```